wordpress-mobile/WordPress-iOS
# New Reader: accessibility improvements

Adds remaining accessibility labels and idenitifiers
ref  #13580

To test:
- Go to the reader, launch the accessibility inspector and verify that all the elements are accessible

PR submission checklist:

- [x] ~~I have considered adding unit tests where possible.~~ Added in a separate PR.
